packages/protocol/contracts/tokenvault/BaseVault.sol

```diff
@@ -16,6 +16,8 @@ abstract contract BaseVault is
     IMessageInvocable,
     IERC165Upgradeable
 {
+    bytes4 internal constant IERC165_INTERFACE_ID = bytes4(keccak256("supportsInterface(bytes4)"));
```

Comment:
do we really need this?

Reply:
how about other contracts that implement IERC165Upgradeable?

Reply:
Yeah i dont have hard opinion. Functional wise, no.